src/styles.css

```diff
@@ -2,15 +2,62 @@
   font-family: sans-serif;
   text-align: center;
 }
-.homeview-user-names    {
-    font-family: Arial, Helvetica, sans-serif;
-    text-transform: capitalize;
+.homeview-user-names {
```

Comment:
Try your best to stick to BEM notation.
What is the component here? 
What is the sub-component?

BEM goes by: component-name__sub-component-name

Reply:
Ah ha, this is from a separate component that I haven't altered, probably showing that I've updated the code because I C&P'd code from another file. From looking at the homeView component, I can see this isn't actually used. I'll work on it in any case!